src/views/Team/components/Hero/Hero.scss

```diff
@@ -0,0 +1,54 @@
+@use "~stylesheets/typography" as tpy;
+@use "~stylesheets/responsiveness" as rsp;
+
+.TeamHero {
+  position: relative;
+  flex-wrap: nowrap;
+  margin: 0;
+  width: 100%;
+  min-height: 35em;
+  padding-top: calc(65px + 10vh);
+  padding-bottom: calc(65px + 10vh);
```

Comment:
I think we could reduce this, especially when we have `margin: 1%` on the container representing the cards.